swagger/fs_wrapper.go

```diff
@@ -0,0 +1,95 @@
+package swagger
+
+import (
+	"bytes"
+	"html/template"
+	"io"
+	"io/fs"
+	"strings"
+	"time"
+)
+
+type fsWrapper struct {
+	fs           fs.FS
+	jsonFileName string
+}
+
+func FsWrapper(fs fs.FS, jsonFilePath string) *fsWrapper {
```

Comment:
This doesn't need to be exported. `newFsWrapper` seems like an adequate name instead.